src/main/java/com/ibm/eventstreams/connect/jdbcsink/database/builder/CommandBuilder.java

```diff
@@ -0,0 +1,172 @@
+package com.ibm.eventstreams.connect.jdbcsink.database.builder;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.kafka.connect.data.Schema;
+
+import com.ibm.eventstreams.connect.jdbcsink.database.datasource.IDataSource;
+import com.ibm.eventstreams.connect.jdbcsink.database.writer.JDBCWriter;
+
+public class CommandBuilder {
+
+    public IDataSource dataSource = null;
+
+    public CommandBuilder(IDataSource connection) {
```

Comment:
These names are not quite lined up. Either the object is a data source or it is a connection to the data source? Would it be possible to clarify this?

Reply:
I have split out class into command builder and database factory which makes it having individual behavior.